trestle/core/commands/author/ssp.py

```diff
@@ -124,89 +108,62 @@ def _init_arguments(self) -> None:
-                        if param.select.choice is not None:
-                            param_text += str(param.select.choice)
-                        param_text = f'[{param_text}]'
-                    # else use the label
-                    if param.label is not None:
-                        param_text = f'[{param.id} = {param.label}]'
-                text = text.replace(param.id, param_text)
-
-        # strip {{ }}
-        text = text.replace(' {{', '').replace(' }}', '').replace('insert: param, ', '').strip()
-
-        return text
-
     def _wrap_label(self, label: str):
-        l_side = '['
-        r_side = ']'
+        l_side = '\['
+        r_side = '\]'
         wrapped = '' if label == '' else f'{l_side}{label}{r_side}'
         return wrapped
 
     def _get_label(self, part: common.Part) -> str:
+        # get the label from the props of a part
         if part.props is not None:
             for prop in part.props:
                 if prop.name == 'label':
                     return prop.value.strip()
         return ''
 
     def _get_part(self, control: cat.Control, part: common.Part) -> List[Union[str, List[str]]]:
+        # for a part in a control find the parts in it that require implementations
+        # return list of string formatted labels and associated descriptive prose
         items = []
-        if part.prose is not None:
-            fixed_prose = self._replace_params(part.prose, control, self._param_dict)
-            label = self._get_label(part)
-            wrapped_label = self._wrap_label(label)
-            pad = '' if wrapped_label == '' else ' '
-            items.append(f'{wrapped_label}{pad}{fixed_prose}')
-        if part.parts is not None:
-            sub_list = []
-            for prt in part.parts:
-                sub_list.extend(self._get_part(control, prt))
-            sub_list.append('')
-            items.append(sub_list)
+        # parts that are sections are output separately
+        if part.name not in self._sections:
+            if part.prose is not None:
+                label = self._get_label(part)
+                wrapped_label = self._wrap_label(label)
+                pad = '' if wrapped_label == '' else ' '
+                items.append(f'{wrapped_label}{pad}{part.prose}')
+            if part.parts is not None:
+                sub_list = []
+                for prt in part.parts:
+                    sub_list.extend(self._get_part(control, prt))
+                sub_list.append('')
+                items.append(sub_list)
         return items
 
     def _add_parts(self, control: cat.Control) -> None:
+        # for a given control add its parts to the md file after replacing params
         items = []
         if control.parts is not None:
             for part in control.parts:
+                # parts that are sections are output separately
+                if part.name in self._sections:
+                    continue
                 if part.name == 'statement':
```

Comment:
See issue: #703 and document as appropriate (this was just the trigger point).

Reply:
The wording of #703 is very odd and hard to parse.  Please clarify there what you are asking for.  As it is, yes - the ssp generation is looking for 'statement'.  I don't know if other things are needed or how it related to FedRAMP.

Reply:
I've re-written issue #703 with some sleep and attention.  TLDR: If we make an assumption that we believe is anchored in 800-53 we should label it as such.

Reply:
Will resolve for now.